README.md

```diff
@@ -18,39 +17,30 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Get started with the linter files and ensure they are present in your repository. 
+Ensure to use the Microverse README template given. 
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
+Get the linter files for HTML and CSS ready in your repository
 
+### Setup
+Ensure you have a VSCode installed in your machine for ease of use
+Get the linter files set up for checking the errors in your code
 ### Install
```

Comment:
- [ ] Kindly be noticed that you should write the steps to install this repo locally like the user should run `git clone https://github.com/Marlyn-Mayienga/Hello_World.git` in the terminal.